### PR TITLE
Bump runtime spec version to 281

### PR DESCRIPTION
**Pull request by nvengal**
Adds CollatorRegistration to parachain staking config

### CODE REVIEW

**Review by lreesby — APPROVED** [review]
lgtm

File: runtime/neumann/src/lib.rs [hunk 1]
```diff
@@ -143,7 +143,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("neumann"),
 	impl_name: create_runtime_str!("neumann"),
 	authoring_version: 1,
-	spec_version: 280,
+	spec_version: 281,
```
Comment (nvengal) [inline runtime/neumann/src/lib.rs:146]: Not sure if there are other versions that need to bumped alongside this

**Review by justinzhou93 — APPROVED** [review]
remember to add to the changelog release notes! should be in a section called "Unreleased". also for future, i wonder if it's better to use git tags rather than branches, since branches can desync, whereas tags don't change

File: runtime/neumann/src/lib.rs (hunk 1 above)
Comment (irsal) [reply runtime/neumann/src/lib.rs:146]: `transaction_version` i think

**Review by justinzhou93 — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```
[the remark this answers is not in the document]
Comment (nvengal) [reply CHANGELOG.md:7]: can do

**Review by justinzhou93 — APPROVED** [review]
🙏  thanks for the changes to all the nit picks haha

**Pull request merged**